go/kbnm/windows/kbnm.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Any way to generate this during build from the template? It's already out of date. :/

If not, need to update it to https://github.com/keybase/client/blob/19c972b13c082b837346996e5af7c46fae5ebae8/go/kbnm/host_json.template which includes the upcoming prod extension id.